fastMCKalman/MC/fastTracker.h

```diff
@@ -66,6 +67,25 @@ Double_t fastTracker::makeSnp(Double_t x1,Double_t y1, Double_t x2,Double_t y2,
   return x0;
 }
 
+Double_t fastTracker::makeYC(Double_t x1,Double_t y1, Double_t x2,Double_t y2, Double_t x3,Double_t y3){
+  //-----------------------------------------------------------------
```

Comment:
To add here:
Used only to check point consistency, not used for the seeding itself